### DISCUSSION TITLE
Vercel app works on vercels domain but not for my custom domain in some countries

### DISCUSSION THREAD

**Discussion by WorkMender** [vercel/next.js#63034] [Help]
### Summary

My custom domain (www.workmender.com) works in Australia. However, I have a few users in Africa and they are unable to access the website. But they can access the vercels domain (https://workmender-client-git-master-workmenders-projects.vercel.app/)
![Whatsapperror](https://github.com/vercel/next.js/assets/156523391/c22de051-ce21-43f4-9b71-2b87711ae3f8)

**Comment by adedotvn** [comment]
Did you find an answer to this?

I am having the same problem with my website.
